### PR TITLE
docs: extend the getVersion documentation

### LINKED ISSUE #1411: getVersion result contains a build number
## Summary

|             |     |
| ----------- | --- |
| Version     | v8.4.3   |
| Affected OS | iOS/Android   |
| OS Version  | ?   |

## Current behavior

It seems sometimes, `getVersion` returns a build number additionally to a version (e.g. 1.9.0+27 or 1.9.0 (27)). I did not catch it by hand, but I found sentry releases with that names while an error report is configured as `release: DeviceInfo.getVersion()`.

I can not present some exact data, but I can list data from sentry:
- Android 6.0.1 Nexus 5X
- Android 12 M2101K7AG
- Android 11 SM-A505FN
- iOS 15.2 iPhone XR
- iOS 15.2.1 iPhone 11
- iOS 15.3.1 iPhone 11
- iOS 15.4.1 iPhone 11
- iOS 14.3 iPhone X
- iOS 15.2 iPhone X
- iOS 15.3.1 iPhone 12

each of devices is not emulator

## Expected behavior

A version from `DeviceInfo.getVersion()` does not contain a build number.

### PR DESCRIPTION
**Pull request by mtnt**
## Description

Fixes #1411

## Checklist

* [ ] I have tested this on a device/simulator for each compatible OS
* [X] I added the documentation in `README.md`
* [ ] I updated the typings files (`privateTypes.ts`, `types.ts`)
* [ ] I added a sample use of the API (`example/App.js`)

### CODE REVIEW

**Review by schie — APPROVED** [review]

**Review by schie — CHANGES_REQUESTED** [review]
sorry, Fat-fingered 'approve' option

File: extendGetVersionDoc [hunk 1]
```diff
@@ -1087,13 +1087,14 @@ DeviceInfo.getUserAgent().then((userAgent) => {
 ### getVersion()
 
 Gets the application version.
+Take in account that a version string is a device/OS formatted and can contains any additional data (such as build number etc.). If you want to be sure about version format, you can use some regexp.
 
 #### Examples
 
 ```js
 let version = DeviceInfo.getVersion();
```
Comment (PhaseboyTA) [inline extendGetVersionDoc:1095]: ) L4

**Review by PhaseboyTA — COMMENTED** [review]
To go or not to go..

**Review by mikehardy — APPROVED** [review]
I just committed @schie suggestion and now this looks good to me, thanks!

**Comment by jbyj** [comment]
@mikehardy Did you intend to nuke the readme as this renames it to [extendGetVersionDoc](https://github.com/react-native-device-info/react-native-device-info/pull/1412/files/ae351997afbf3321732d1cff3e8466705d27dd5c#diff-e58800435b2e23b8d660db9dad2e10c5d8e3ca4f93f5208495424b21786cc07c)?

**Comment by mikehardy** [comment]
Oh 💩 ! Nope. Thanks for checking, I'm out at lunch now but will look after

**Comment by mikehardy** [comment]
Fixed, thanks again @jbyj

**Pull request merged**